src/__tests__/views/article.spec.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Article from '../../views/Article/Article';
+
+describe('Article', () => {
+  const mockStore = configureStore([thunk]);
+  let component;
+  let article = {
+    title: 'test',
+    description: 'description',
+    images: ['image'],
+    body: 'body',
+  };
+
+  it('renders correctly', () => {
+    const store = mockStore({
+      articleReducer: {
+        article,
+      },
+    });
+
+    component = mount(
+      <Provider store={store}>
+        <Article
+          history={{ location: { pathname: '/articles/:slug' } }}
+          match={{ params: { slug: 'poierwjvoejcijwoei' } }}
+        />
+      </Provider>
+    );
+    expect(component.find('.article-container').exists()).toBe(true);
+  });
+
+
+  it('renders spinner when article == null', () => {
```

Comment:
@Dubby20 same thing here. What condition makes article null? That should be what you're testing.